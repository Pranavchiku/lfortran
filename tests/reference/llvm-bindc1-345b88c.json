{
    "basename": "llvm-bindc1-345b88c",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/bindc1.f90",
    "infile_hash": "b894bae3cff84c3540cc34d38c5e3e6c46197db6d93d5f0b48013a06",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-bindc1-345b88c.stdout",
    "stdout_hash": "e759c034a43cb6091acc450f13d788d507999a761e8d786fdbd14a6e",
    "stderr": "llvm-bindc1-345b88c.stderr",
    "stderr_hash": "9e20d031e27e2b48685d70be45dce001d004dc3c45d4e073c4969766",
    "returncode": 0
}
