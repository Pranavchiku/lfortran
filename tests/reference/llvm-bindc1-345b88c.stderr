warning: Implied save feature
 --> tests/../integration_tests/bindc1.f90:3:1
  |
3 | integer, pointer :: x
  | ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/bindc1.f90:4:1
  |
4 | type(c_ptr) :: p
  | ^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
